### DISCUSSION TITLE
add icon to navbar

### DISCUSSION THREAD

**Discussion by sharplabBIU** [alshedivat/al-folio#2371] [Q&A] [answered]
Hi, is there an easy way to add the icon defined in assets/img in the config file to the far-left of the navbar?

thank you so much for any time taken to help!
paul

**Comment by george-gca** [comment, marked as answer]
[Here](https://github.com/alshedivat/al-folio/blob/492c50964683921299502e3765d36dc227b56436/_includes/head.liquid#L65) is where the icon is set in `_includes/head.liquid`. Note that we access it via `site.icon` and do some modification to its `href` to correct the path. The header of the site is defined in [_includes/header.liquid](https://github.com/alshedivat/al-folio/blob/492c50964683921299502e3765d36dc227b56436/_includes/header.liquid), which is where you'll want to add the icon.

**Reply by sharplabBIU** [reply]
thank you so much for taking the time to assist!
